designer/server/src/main/scala/pl/touk/nussknacker/ui/validation/UIProcessValidator.scala

```diff
@@ -134,7 +133,7 @@ class ProcessValidation(
         .map(modelCategoryValidator.withExpressionParsers)
         .getOrElse(modelCategoryValidator)
     }
-    // TODO: should we validate after resolving?
+    // TODO: should we validate after resolve?
```

Comment:
I'm not sure I understand this comment - this asks if we should validate after resolving fragment? Or resolving to canonical process or resolving expressions? IMO this could ideally be phrased as "should we validate after resolving [framgent or smth] / [fragment/smth] resolution"

Reply:
I'm not the author of this comment but for me, it looks like someone has reasonable concerns that validation should be done after fragment resolution, not before it to collect also errors related with this resolution. Unfortunately, author of this comment didn't leave us a notice, why he/she didn't do this in the proposed order.